### ISSUE TITLE
Old version of JSLint

### ISSUE THREAD

**Issue by jt55401** [muhqu/jslint-plugin-for-jedit#3]
Looks like this is now a very dated version of JSLint. Would be great if an updated version could be included in a future version of this plugin.

**Comment by tomvantilburg** [comment]
I agree, it would be great to be able to reconfigure it to use a new jslinter.
In the .jar file there is a full jslint.js available. I tried replacing it with the newest jslint.js but the plugin doesn't do anything after that.
